src/number-input/__test__/NumberInput.test.tsx

```diff
@@ -69,37 +72,35 @@ describe("NumberInput", () => {
 
     const incBtn = testId("inc");
     const decBtn = testId("dec");
+    const numberInput = testId("numberinput");
 
-    expect(testId("numberinput")).not.toHaveFocus();
+    expect(numberInput).not.toHaveFocus();
     press.Tab();
-    expect(testId("numberinput")).toHaveFocus();
-    expect(testId("numberinput")).toHaveValue("0");
-    click(incBtn);
-    click(incBtn);
-    click(incBtn);
-    expect(testId("numberinput")).toHaveValue("3");
-    click(decBtn);
-    click(decBtn);
-    click(decBtn);
-    expect(testId("numberinput")).toHaveValue("0");
+    expect(numberInput).toHaveFocus();
+    expect(numberInput).toHaveValue("0");
+    repeat(() => click(incBtn), 3);
+    expect(numberInput).toHaveValue("3");
+    repeat(() => click(decBtn), 3);
+    expect(numberInput).toHaveValue("0");
   });
 
   it("should increase/decrease with scrollwheel", () => {
     const { getByTestId: testId } = render(
       <NumberInputComp defaultValue={0} />,
     );
+    const numberInput = testId("numberinput");
 
     press.Tab();
-    expect(testId("numberinput")).toHaveFocus();
-    expect(testId("numberinput")).toHaveValue("0");
+    expect(numberInput).toHaveFocus();
+    expect(numberInput).toHaveValue("0");
 
-    fireEvent.wheel(testId("numberinput"), { deltaY: -100 });
-    fireEvent.wheel(testId("numberinput"), { deltaY: -100 });
-    expect(testId("numberinput")).toHaveValue("2");
+    fireEvent.wheel(numberInput, { deltaY: -100 });
+    fireEvent.wheel(numberInput, { deltaY: -100 });
+    expect(numberInput).toHaveValue("2");
 
-    fireEvent.wheel(testId("numberinput"), { deltaY: 100 });
-    fireEvent.wheel(testId("numberinput"), { deltaY: 100 });
-    expect(testId("numberinput")).toHaveValue("0");
+    fireEvent.wheel(numberInput, { deltaY: 100 });
+    fireEvent.wheel(numberInput, { deltaY: 100 });
+    expect(numberInput).toHaveValue("0");
   });
```

Comment:
That moment when your brain forgets that javascript has something called "variables"